.test_ignore.txt

```diff
@@ -0,0 +1,16 @@
+github.com/gittuf/gittuf/docs/cli
+github.com/gittuf/gittuf/internal/cmd/addhooks
+github.com/gittuf/gittuf/internal/cmd/clone
+github.com/gittuf/gittuf/internal/cmd/dev
+github.com/gittuf/gittuf/internal/cmd/policy
+github.com/gittuf/gittuf/internal/cmd/profile
+github.com/gittuf/gittuf/internal/cmd/root
+github.com/gittuf/gittuf/internal/cmd/rsl
+github.com/gittuf/gittuf/internal/cmd/trust
+github.com/gittuf/gittuf/internal/cmd/verifycommit
+github.com/gittuf/gittuf/internal/cmd/verifyref
+github.com/gittuf/gittuf/internal/cmd/verifytag
+github.com/gittuf/gittuf/internal/cmd/version
```

Comment:
Nit

```suggestion
github.com/gittuf/gittuf/docs/
github.com/gittuf/gittuf/internal/cmd
```

Reply:
github.com/gittuf/gittuf/internal/cmd/common has tests

Reply:
Thanks, I did not see that.